src/libraries/System.Security.Cryptography.ProtectedData/src/System/Security/Cryptography/ProtectedData.cs

```diff
@@ -19,7 +22,148 @@ public static byte[] Protect(byte[] userData, byte[]? optionalEntropy, DataProte
             if (userData is null)
                 throw new ArgumentNullException(nameof(userData));
 
-            return ProtectOrUnprotect(userData, optionalEntropy, scope, protect: true);
+            byte[]? outputData;
+            bool result = TryProtectOrUnprotect(
+                userData,
+                optionalEntropy,
+                scope,
+                protect: true,
+                allocateArray: true,
+                bytesWritten: out _,
+                outputData: out outputData
+            );
+            Debug.Assert(result);
```

Comment:
```suggestion
                outputData: out outputData);

            Debug.Assert(result);
```

I believe the prevailing style is close-paren on the same line as the last argument, and then a blank line after (unless followed by `}`)

(repeats)